app/serializers/api/public/v1/serializable_location_status.rb

```diff
@@ -9,6 +9,12 @@ class SerializableLocationStatus < JSONAPI::Serializable::Resource
         attributes :publish,
                    :status
 
+        belongs_to :location do
+          data do
+            @object.site
+          end
+        end
+
```

Comment:
Documentation needs updating

Reply:
I am now using another end point to fetch locations - changes no longer required